sdk/signalr/Microsoft.Azure.WebJobs.Extensions.SignalRService/README.md

```diff
@@ -0,0 +1,123 @@
+# Azure Functions Bindings for Azure SignalR Service
+
+## Build Status
+
+Travis: [![travis](https://travis-ci.org/Azure/azure-functions-signalrservice-extension.svg?branch=dev)](https://travis-ci.org/Azure/azure-functions-signalrservice-extension)
```

Comment:
Should this be removed?

Reply:
I will try to rewrite the readme in the format of ServiceBus.